integration-tests/postgres_test.go

```diff
@@ -88,7 +88,7 @@ var _ = Describe("postgres", func() {
 				err := broker.Update(instanceGUID, "csb-google-postgres", postgresNoOverridesPlan["name"].(string), map[string]interface{}{key: value})
 				Expect(err).NotTo(HaveOccurred())
 			},
-			Entry("cores", "cores", 5),
+			Entry("tier", "tier", "db-g1-small"),
 			Entry("storage_gb", "storage_gb", 11),
```

Comment:
**thoughts (no-blocking)**: It seems that the `db-g1-small` tier is related to 20GB storage. This test is about changing some properties, and I understand that the operation is allowed. I wondered if  20GB is the maximum value when it comes to managing the available space, if so, maybe it would be nice to add it to the description.

Reply:
Hi @zucchinidev - looking at the google documentation I think the `db-g1-small` tier can be extended up to 3TiB of disk space. 
For the description of this property I added that it determines CPU/RAM given that we have a separate field for defining the disk size. 
You are right that we need to make it clear to users to read the Google documentation to fully understand what a service tier affects.

Reply:
Hi @jimbo459 - Thank you very much for your reply and thank you for the link to the documentation!